ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -56,7 +59,12 @@ public:
   using LocalView = typename Traits::LocalView;
   using Geometry  = typename Traits::Geometry;
   using GridView  = typename Traits::GridView;
-  using Pre       = EnhancedAssumedStrainsPre;
+  using Pre       = EnhancedAssumedStrainsPre<ES>;
+
+  static constexpr auto enhancedStrain = ES;
```

Comment:
Is the enhancedStrain type needed as a templated should this be deduced from the `underlying` object?
In the member functions. Because also below the function `eas(...)` now also has a template paramter which has to be artificially synced with the one used by the displacementbased elment.